x-pack/test/apm_api_integration/tests/time_range_metadata/time_range_metadata.spec.ts

```diff
@@ -82,24 +79,25 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       describe('data loaded with and without summary field', () => {
         const localStart = moment('2023-04-28T00:00:00.000Z');
         const localEnd = moment('2023-04-28T06:00:00.000Z');
+
         before(async () => {
           const regularData = getTransactionEvents(localStart, localEnd);
-          await synthtraceEsClient.index([...regularData]);
+          await synthtraceEsClient.index(regularData);
+
           const { previousStart, previousEnd } = subtractDateDifference(localStart, localEnd);
-          const previousDataWithoutSummaryField = getTransactionEvents(previousStart, previousEnd);
-          synthtraceEsClient.pipeline(
-            overwriteSynthPipelineWithSummaryFieldDeleteTransform({
-              synthtraceEsClient,
-            })
-          );
-          await synthtraceEsClient.index([...previousDataWithoutSummaryField]);
+          const previousData = getTransactionEvents(previousStart, previousEnd);
+
+          const transforms = [deleteSummaryFieldTransform()];
+          // FIX: this casues a timeout in the test. Why??
+          await synthtraceEsClient.index(previousData, transforms);
```

Comment:
@dgieselaar Any idea why this causes the test to timeout with

```
 Error: Timeout of 360000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/sqren/elastic/kibana/x-pack/test/apm_api_integration/tests/index.ts)
```

Removing `transforms` solves the timeout (the test then fails because the transforms are needed).

```suggestion
          await synthtraceEsClient.index(previousData);
```